.jsdoc.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   source: {
     excludePattern: '(^|\\/|\\\\)[._]',
     include: [
-      'src'
+      './build/src'
```

Comment:
we should potentially pull `.jsdoc` into our generator and out of `synthtool`, so that we can properly populate this path.

Reply:
Agreed - we will do it.

Reply:
We still want to keep a consistent jsdoc file for non-gapic generated repos too!

Reply:
https://github.com/googleapis/gapic-generator-typescript/issues/92